src/SDKs/DataFactory/Management.DataFactory/Generated/Models/ValidationActivity.cs

```diff
@@ -1,127 +0,0 @@
-            CustomInit();
-        }
-
-        /// <summary>
-        /// Initializes a new instance of the ValidationActivity class.
-        /// </summary>
-        /// <param name="name">Activity name.</param>
-        /// <param name="dataset">Validation activity dataset
-        /// reference.</param>
-        /// <param name="additionalProperties">Unmatched properties from the
-        /// message are deserialized this collection</param>
-        /// <param name="description">Activity description.</param>
-        /// <param name="dependsOn">Activity depends on condition.</param>
-        /// <param name="userProperties">Activity user properties.</param>
-        /// <param name="timeout">Specifies the timeout for the activity to
-        /// run. If there is no value specified, it takes the value of
-        /// TimeSpan.FromDays(7) which is 1 week as default. Type: string (or
-        /// Expression with resultType string), pattern:
-        /// ((\d+)\.)?(\d\d):(60|([0-5][0-9])):(60|([0-5][0-9])). Type: string
-        /// (or Expression with resultType string), pattern:
-        /// ((\d+)\.)?(\d\d):(60|([0-5][0-9])):(60|([0-5][0-9])).</param>
-        /// <param name="sleep">A delay in seconds between validation attempts.
-        /// If no value is specified, 10 seconds will be used as the
-        /// default.</param>
-        /// <param name="minimumSize">Minimum size of a file in byte. If no
-        /// value is specified, 0 byte will be used as the default.</param>
-        public ValidationActivity(string name, DatasetReference dataset, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), string description = default(string), IList<ActivityDependency> dependsOn = default(IList<ActivityDependency>), IList<UserProperty> userProperties = default(IList<UserProperty>), object timeout = default(object), int? sleep = default(int?), int? minimumSize = default(int?))
-            : base(name, additionalProperties, description, dependsOn, userProperties)
-        {
-            Timeout = timeout;
-            Sleep = sleep;
-            MinimumSize = minimumSize;
-            Dataset = dataset;
-            CustomInit();
-        }
-
-        /// <summary>
-        /// An initialization method that performs custom operations like setting defaults
-        /// </summary>
-        partial void CustomInit();
-
-        /// <summary>
-        /// Gets or sets specifies the timeout for the activity to run. If
-        /// there is no value specified, it takes the value of
-        /// TimeSpan.FromDays(7) which is 1 week as default. Type: string (or
-        /// Expression with resultType string), pattern:
-        /// ((\d+)\.)?(\d\d):(60|([0-5][0-9])):(60|([0-5][0-9])). Type: string
-        /// (or Expression with resultType string), pattern:
-        /// ((\d+)\.)?(\d\d):(60|([0-5][0-9])):(60|([0-5][0-9])).
-        /// </summary>
-        [JsonProperty(PropertyName = "typeProperties.timeout")]
```

Comment:
Please note that this is a breaking change

Reply:
@dsgouda  This changes hasn't been released yet. We had similar comments on Specs PR and got sign off from them and ARM team. These are new activities we added last week, but didn't want them in this release, so we took them out